artifactory/services/utils/artifactoryutils.go

```diff
@@ -239,6 +241,47 @@ func createBodyForLatestBuildRequest(buildName, buildNumber string) (body []byte
 	return
 }
 
+func filterAqlSearchResultsByBuildSaveToFile(specFile *ArtifactoryCommonParams, cr *content.ContentReader, flags CommonConf, itemsAlreadyContainProperties bool) (*content.ContentReader, error) {
```

Comment:
Let's rename this function to *filterAqlSearchResultsByBuild* and remove the old *filterAqlSearchResultsByBuild* function.